File: app/src/main/java/com/ilustris/motiv/beans/Pics.java
package com.ilustris.motiv.beans;

public class Pics {

    public Pics() {
    }

    public Pics(String uri, String id, String name) {
        this.uri = uri;
        this.id = id;
        this.name = name;
    }

    public String getUri() {
        return uri;
    }

    public void setUri(String uri) {
        this.uri = uri;
    }

    private String uri;
    private String id;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    private String name;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }
}

File: app/src/main/java/com/ilustris/motiv/beans/Pics.kt
package com.ilustris.motiv.beans

class Pics {

    var uri: String? = null
    var id: String? = null

    var name: String? = null

    constructor() {}

    constructor(uri: String, id: String, name: String) {
        this.uri = uri
        this.id = id
        this.name = name
    }
}
